crates/astria-composer/src/executor/mod.rs

```diff
@@ -415,6 +419,23 @@ impl Executor {
 
         reason.map(|_| ())
     }
+
+    // check for mismatched configured chain_id and sequencer client chain_id
+    pub(crate) async fn check_chain_ids(&self) -> eyre::Result<()> {
+        let client_response = self
+            .sequencer_client
+            .status()
+            .await
```

Comment:
This future should be wrapped in a retry because composer is likely to start way earlier then sequencer and cometbft. This means that you will a bunch of crash-look-backoff errors in kubernetes.

See conductor for doing essentially the same thing: https://github.com/astriaorg/astria/blob/c31c10b6db41b19fdce8e4b172363f855d88be23/crates/astria-conductor/src/celestia/mod.rs#L645

Reply:
Done in f3c96ff.